__tests__/app.test.js

```diff
@@ -136,6 +136,68 @@ describe('GET/api/reviews/:review_id', () => {
 
 ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
+describe('GET /api/reviews/:review_id/comments', () => {
+  test('return an array of comments for the given review_id with each comment having the following properties: comment_id, votes, created_at, author, body, review_id', () => {
+    return request(app)
+      .get('/api/reviews/3/comments')
+      .expect(200)
+      .then(({body}) => {
+        expect(body).toBeSortedBy("created_at", { descending: true })              
+        expect(body).toBeInstanceOf(Array)        
+        expect(body).toHaveLength(3)        
+        body.forEach((comment) => {
+          expect(comment).toEqual(
+            expect.objectContaining({               
+              comment_id: expect.any(Number),             
+              votes: expect.any(Number),
+              created_at: expect.any(String),
+              author: expect.any(String),
+              body: expect.any(String),
+              review_id: expect.any(Number)
+            })
+          )
+        })
+      })
+  })
+
+
+test('return "status:404, Route not found"', () => {
+  return request(app)
+    .get('/api/someRandomThingThatIsntAValidPath')
+    .expect(404)
+    .then(({body}) => {
+      expect(body.msg).toBe("Route not found")
+
+  return request(app)
+    .get('/someRandomThingThatIsntAValidPath')
+    .expect(404)
+    .then(({body}) => {
+      expect(body.msg).toBe("Route not found")
+    })
+  })      
+})
+
+test('return "status:404, sorry, there are no comments for that review, or there is no review with that id" when asked for a review_id that doesnt exist or the review has no comments', () => {
+  return request(app)
+    .get('/api/reviews/100000/comments')
+    .expect(404)
+    .then(({body}) => {
+      expect(body.msg).toBe("sorry, there are no comments for that review, or there is no review with that id")
```

Comment:
I'd suggest having a 404 only for if the review id isn't found ... if there are no comments yet, return a 200 with an empty array